src/modules/iasql_functions/iasql.ts

```diff
@@ -102,6 +102,7 @@ export async function install(
   syncContext?: Context,
   ormOpt?: TypeormWrapper,
 ) {
+  const loggerWithUser = logger.scope({ dbUser });
```

Comment:
hmm I was hoping this would be per user using the auth0 id like telemetry. I originally did telemetry per db, but the drawback with this approach is that you cannot see all the interactions per user since disconnects and connects won't show up and form different users entirely so I won't be able to ask of mezmo to truly show me all logs for a specific user